### PR TITLE
Docker Setup

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

**Comment by NileshPant1999** [comment]
@ruuushhh make sure to add this to integrations central repo too

**Review by NileshPant1999 — APPROVED** [review]

File: centralized_run.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Run db migrations
+python manage.py migrate
+
+# Creating the cache table
+python manage.py createcachetable --database cache_db
+
+# Running development server
+gunicorn -c gunicorn_config.py sage_desktop_api.wsgi -b 0.0.0.0:8009
```
Comment (NileshPant1999) [inline centralized_run.sh:10]: this is wrong

File: init.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+CREATE DATABASE sage_desktop_db;
```
Comment (NileshPant1999) [inline init.sql:1]: this is wrong

File: run.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Run db migrations
+python manage.py migrate
+
+# Creating the cache table
+python manage.py createcachetable --database cache_db
+
+# Running development server
+gunicorn -c gunicorn_config.py sage_desktop_api.wsgi -b 0.0.0.0:8000
```
Comment (NileshPant1999) [inline run.sh:10]: path is wrong

**Review by Shwetabhk — APPROVED** [review]

**Pull request merged**